From 77ccc1f2e41106bca6890ce0340b93714c7cc51f Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Mon, 06 Aug 2018 13:49:11 +0300
Subject: [PATCH] Add dali packets support fixup

When flash is erased we gets "ff" this is breaking -ne
check as string and use "=" operator
---

diff --git a/board/habanalabs/common/rootfs_overlay/usr/sbin/flash_burn b/board/habanalabs/common/rootfs_overlay/usr/sbin/flash_burn
index f699c11..ea59d88 100755
--- a/board/habanalabs/common/rootfs_overlay/usr/sbin/flash_burn
+++ b/board/habanalabs/common/rootfs_overlay/usr/sbin/flash_burn
@@ -65,7 +65,7 @@
 # 5 -- FW 1
 STABLE_IMAGE_CHAR=`xxd -p -l1 -c1 /dev/mtd2`
 # '1' / '2' equal (ascii) 0x31 / 0x32
-if [ "${STABLE_IMAGE_CHAR}" -ne "31" ] && [ "${STABLE_IMAGE_CHAR}" -ne "32" ]; then
+if [ ! "${STABLE_IMAGE_CHAR}" = "31" ] && [ ! "${STABLE_IMAGE_CHAR}" = "32" ]; then
 	STABLE_IMAGE_CHAR=31
 fi
 STABLE_IMAGE_NUM=$((0x${STABLE_IMAGE_CHAR} - 0x31))
